puzzle.japan.txt

```diff
@@ -78,121 +78,121 @@ element.keychest_skeleton.5	single(smallskeletonarmorb:
-prop.red_fountain.12	が出た。
+prop.red_fountain.12	が出た.
 prop.red_fountain.13	
-prop.red_fountain.14	が出た。
+prop.red_fountain.14	が出た.
 prop.red_fountain.15	
-prop.red_fountain.16	が出た。
+prop.red_fountain.16	が出た.
 prop.red_fountain.17	
-prop.red_fountain.18	が出た。
+prop.red_fountain.18	が出た.
 prop.red_fountain.19	
-prop.red_fountain.20	が出た。
+prop.red_fountain.20	が出た.
 prop.red_fountain.21	
-prop.red_fountain.22	が出た。
+prop.red_fountain.22	が出た.
 prop.red_fountain.23	
-prop.red_fountain.24	が出た。
+prop.red_fountain.24	が出た.
 prop.red_fountain.25	
-prop.red_fountain.26	が出た。
+prop.red_fountain.26	が出た.
 prop.red_fountain.27	
-prop.red_fountain.28	が出た。
-prop.red_fountain.29	赤い噴水
+prop.red_fountain.28	が出た.
+prop.red_fountain.29	Red fountain
 prop.red_fountain.30	の効果で
-prop.red_fountain.31	の効果が消えていく。
-prop.red_fountain.32	の効果が消えた。
+prop.red_fountain.31	の効果が消えていく.
+prop.red_fountain.32	の効果が消えた.
 prop.red_fountain.33	
-prop.red_fountain.34	が出た。
+prop.red_fountain.34	が出た.
 prop.red_fountain.35	
-prop.red_fountain.36	が出た。
+prop.red_fountain.36	が出た.
 prop.red_fountain.37	
-prop.red_fountain.38	が出た。
+prop.red_fountain.38	が出た.
 prop.red_fountain.39	
-prop.red_fountain.40	が出た。
+prop.red_fountain.40	が出た.
 prop.red_fountain.41	
-prop.red_fountain.42	が出た。
+prop.red_fountain.42	が出た.
 prop.red_fountain.43	
-prop.red_fountain.44	が出た。
+prop.red_fountain.44	が出た.
 prop.red_fountain.45	
-prop.red_fountain.46	が出た。
+prop.red_fountain.46	が出た.
 prop.red_fountain.47	
-prop.red_fountain.48	が出た。
+prop.red_fountain.48	が出た.
 prop.red_fountain.49	
-prop.red_fountain.50	が出た。
+prop.red_fountain.50	が出た.
 prop.red_fountain.51	
-prop.red_fountain.52	が出た。
+prop.red_fountain.52	が出た.
 prop.red_fountain.53	
-prop.red_fountain.54	が出た。
+prop.red_fountain.54	が出た.
 prop.red_fountain.55	
-prop.red_fountain.56	が出た。
-prop.red_fountain.57	赤い噴水の効果で {0} が出た。
-prop.red_fountain.58	赤い噴水の効果が消えていく。
-prop.red_fountain.59	赤い噴水の効果が消えた。
-prop.red_fountain.60	赤い噴水の効果で {0} が出た。
-prop.red_fountain.61	赤い噴水の効果が消えていく。
-prop.red_fountain.62	赤い噴水の効果が消えた。
-prop.tirnanog_sealstone.1	ユーザーがティルナノイの封印に触れた。
-prop.tirnanog_sealstone.2	この封印を解く準備が出来ていないようだ。
-prop.tirnanog_sealstone.3	クリステルの話によると、\n封印の力が弱くなるサーオィンにならないと封印を解くことはできないそうだ。
-prop.tirnanog_sealstone.4	弱すぎて、門が開かないようだ。
-prop.tirnanog_sealstone.7	私に解ける封印ではない。\nパーティーのリーダーなら解けそうだ。
-prop.tirnanog_sealstone.8	封印の門の向こうの世界に移動しますか？
-prop.tirnanog_sealstone.9	レベル25以上の人が集まらないと封印は解けないようだ。私たちの力では無理だ。
-prop.tirnanog_sealstone.10	レベル25以上の人が集まらないと封印は解けないようだ。私たちの力では無理だ。
+prop.red_fountain.56	が出た.
+prop.red_fountain.57	Red fountain gave {0}.
+prop.red_fountain.58	Red fountain effect faded.
+prop.red_fountain.59	Red fountain effect faded.
+prop.red_fountain.60	Red fountain gave {0}.
+prop.red_fountain.61	Red fountain effect faded.
+prop.red_fountain.62	Red fountain effect faded.
+prop.tirnanog_sealstone.1	Player touches the seal of Tir Na Nog.
+prop.tirnanog_sealstone.2	I am not able to break this seal.
+prop.tirnanog_sealstone.3	クリステルの話によると、\n封印の力が弱くなるサーオィンにならないと封印を解くことはできないそうだ.
```

Comment:
"According to Kristell, the seal cannot be broken until Samhain, when the power of the seal becomes weaker."